collection/lucas.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "lucas",
+    "meaning": "Light-giving or illumination",
+    "aliases": [
+      "luke",
+      "luc"
+    ],
+    "translations": {
+        "pol": "Łukasz"
```

Comment:
Hi @tiffwchen,
Can you please update this translation to use lowercase letters? `ł` instead of `Ł`. This should fix the test failing.